src/context/sessionContext/index.tsx

```diff
@@ -129,11 +152,12 @@ export const SessionProvider = ({
     });
   }, [session, sessionId, onError, currentLocation, restoreSession]);
 
-  const contextLogin = async (options: ILoginInputOptions) => {
+  const contextLogin = async (options: Parameters<typeof login>[0]) => {
     setSessionRequestInProgress(true);
 
     try {
       await login(options);
+      setProfile(undefined);
```

Comment:
This is on login, after `setProfile` was called potentially; you want this on `contextLogout`

Reply:
I should try to type things with my eyes open once in a while ><.